services/gitclub/app/fixtures.py

```diff
@@ -73,36 +158,81 @@ def load_fixture_data(session):
     #################
 
     for repo in repos:
-        oso.tell("has_relation", repo, "parent", repo.org)
+        facts.append(["has_relation", { "type": "Repository", "id": str(repo.id)}, "organization", { "type": "Organization", "id": str(repo.org.id) }])
+        facts.append(["is_protected", { "type": "Repository", "id": str(repo.id)}, { "type": "Boolean", "id": str(repo.protected).lower()}])
+        if repo.public:
+            facts.append(["is_public", { "type": "Repository", "id": str(repo.id)}])
 
-    for issue in issues:
-        oso.tell("has_relation", issue, "parent", issue.repo)
+    # for issue in issues:
```

Comment:
This whole section is a TODO for me, I believe.

Reply:
Nah, nothing to do here.